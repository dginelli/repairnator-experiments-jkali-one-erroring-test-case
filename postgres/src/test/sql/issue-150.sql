SELECT v->'top-tags'->'buckets'
FROM zdb_json_aggregate('so_posts', '{ "top-tags":{"terms":{"field":"tags", "size" : 3}}}', 'java') v;