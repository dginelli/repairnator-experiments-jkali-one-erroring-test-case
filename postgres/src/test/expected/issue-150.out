SELECT v->'top-tags'->'buckets'
FROM zdb_json_aggregate('so_posts', '{ "top-tags":{"terms":{"field":"tags", "size" : 3}}}', 'java') v;
                                                 ?column?                                                 
----------------------------------------------------------------------------------------------------------
 [{"key":"java","doc_count":2713},{"key":"design","doc_count":695},{"key":"programming","doc_count":528}]
(1 row)

